rel/i18n/emqx_limiter_schema.hocon

```diff
@@ -24,18 +24,19 @@ max_conn_burst.label:
 messages_rate.desc:
 """Used to limit the number of messages a single client can send to EMQX per second.
 
-Once the limit is reached, EMQX will pause reading data from the receive-buffer, thus slowing down or even temporarily hanging the sender.
+Once the limit is reached, EMQX will drop QoS 0 messages and reject QoS 1 and QoS 2 messages
+with "Rate Exceeded" error code (0x9F).
```

Comment:
I think we shouldn't reply `0x9F` "Connection rate exceeded" for publishing, it seems to be only for `CONNACK` and `DISCONNECT` packets according to the spec:

![image](https://github.com/user-attachments/assets/e2a45539-0dc4-4e43-b52d-55164842c549)

Reply:
Yes, we respond with quota exceeded